pkg/clustermesh/common/clustermesh_test.go

```diff
@@ -256,9 +261,13 @@ func TestClusterMeshMultipleAddRemove(t *testing.T) {
 	block, _ := blockRemoval.Load("cluster1")
 	close(block)
 
-	// Multiple removals and additions, ending with an addition should lead to a ready cluster
+	// A blocked removal following by multiple removals and additions, should lead to a not ready cluster
```

Comment:
The original comment is actually correct, if you consider the entire set of operations until:

```
require.EventuallyWithT(t, func(c *assert.CollectT) { assert.True(c, isReady("cluster2")) }, timeout, tick, "Cluster2 is not ready")
```

And similarly for the one below.

Reply:
Indeed I didn't meant to say the the original comment was wrong but wanted to clarify a bit the flow of the test. But yes this isn't clear now for the entire set of operation now... I ended up keeping the original comment and added an additional one to try to have the best of both world